tlsn/tlsn-prover/src/tls/mod.rs

```diff
@@ -74,7 +74,11 @@ impl Prover<state::Initialized> {
         self,
         socket: S,
     ) -> Result<Prover<state::Setup>, ProverError> {
-        let mut mux = UidYamux::new(yamux::Config::default(), socket, yamux::Mode::Client);
+        let mut mux_config = yamux::Config::default();
+        mux_config.set_max_num_streams(40);
```

Comment:
I feel like we should add a comment here explaining the "magic numbers".
A link to this PR might be OK